src/fu-engine-config.c

```diff
@@ -29,6 +29,19 @@ struct _FuEngineConfig {
 
 G_DEFINE_TYPE(FuEngineConfig, fu_engine_config, FU_TYPE_CONFIG)
 
+/* defaults changed here will also be reflected in the fwupd.conf man page */
+#define FU_DAEMON_CONFIG_DEFAULT_DISABLED_PLUGINS      "test;test_ble"
+#define FU_DAEMON_CONFIG_DEFAULT_URI_SCHEMES	       "file;https;http;ipfs"
+#define FU_DAEMON_CONFIG_DEFAULT_UPDATE_MOTD	       TRUE
+#define FU_DAEMON_CONFIG_DEFAULT_IGNORE_POWER	       FALSE
+#define FU_DAEMON_CONFIG_DEFAULT_ONLY_TRUSTED	       TRUE
+#define FU_DAEMON_CONFIG_DEFAULT_SHOW_DEVICE_PRIVATE   TRUE
+#define FU_DAEMON_CONFIG_DEFAULT_ALLOW_EMULATION       FALSE
+#define FU_DAEMON_CONFIG_DEFAULT_ENUMERATE_ALL_DEVICES TRUE
+#define FU_DAEMON_CONFIG_DEFAULT_TRUSTED_UIDS	       NULL
+#define FU_DAEMON_CONFIG_DEFAULT_HOST_BKC	       NULL
+#define FU_DAEMON_CONFIG_DEFAULT_TRUSTED_REPORTS       "VendorId=$OEM"
```

Comment:
can we perhaps put all the defaults for everything into a single source of truth header?  `fu-engine-config-defaults.h`.  Then the thing that parses it looks at one file and the plugins and engine that use it just have it included.